addons/web/controllers/main.py

```diff
@@ -1814,7 +1814,7 @@ def namelist(self, model, export_id):
 
         return [
             {'name': field['name'], 'label': fields_data[field['name']]}
-            for field in export_fields_list
+            for field in export_fields_list if field['name'] in self.fields_get(model)
```

Comment:
You should put `self.fields_get(model)` in a variable to avoid the call every time